utils/install.py

```diff
@@ -4,13 +4,17 @@
 from urllib.parse import quote_plus
 from urllib.parse import urljoin
 from subprocess import call
+from environs import Env
 
 sys.path.append(os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")))
 
-from utils import user_input, error
+from utils import error
 from fame.common.constants import FAME_ROOT
 from fame.common.pip import pip_install
 
+env = Env()
+env.read_env(env.str('ENV_PATH', '.env'), recurse=False)
```

Comment:
Why use a third party library to read environment variables ?

Reply:
Because env library easily handle default values. 

`context['fame_url'] = env("FAME_URL", "http://localhost")`

If you want, I can also remove this lib and use a lot of "if" instead
```
context['fame_url'] =  'http://localhost'
if 'FAME_URL' in os.environ and os.environ['FAME_URL']:
    context['fame_url'] =  os.environ['FAME_URL']
```

would you like me to do that?

Reply:
You can actually do the same without a library:

```python
os.getenv("FAME_URL", "http://localhost")
```

or

```python
os.environ.get("FAME_URL", "http://localhost")
```